### PR TITLE
Bug/back to dashboard button style fix

### PR DESCRIPTION
**Pull request by bipin-shrestha**
Fixed the buttons "Go to Point Share" and "back to Dashboard"

### CODE REVIEW

File: src/components/pages/Leaderboard/RenderLeaderboard.js [hunk 1]
```diff
@@ -19,13 +19,19 @@ const RenderLeaderboard = props => {
...
-      </Button>
-      <div className="trophy-container">
+      <div className="trophy-container" style={{ marginTop: '1rem' }}>
         <Leaderboard child={props.child} />
         <div className="custom-divider"></div>
       </div>
+      <div style={{ textAlign: 'center' }}>
+        <Button
+          className="back-btn"
+          style={{ margin: '1rem' }}
```
Comment (PVigar88) [inline src/components/pages/Leaderboard/RenderLeaderboard.js:29]: Should we be coding in-line styles?  seems better suited to be in the sass.

File: src/components/pages/MatchUp/RenderMatchUp.js [hunk 2]
```diff
@@ -19,7 +19,15 @@ const RenderMatchUp = props => {
     <>
       <Header displayMenu={true} title="Scribble Stadium" />
       {/**Temporary button to redirect to point share component */}
-      <Button onClick={goToPointShare}>Go To Point Share</Button>
+      <div style={{ textAlign: 'center' }}>
+        <Button
+          style={{ marginTop: '1rem' }}
```
Comment (PVigar88) [inline src/components/pages/MatchUp/RenderMatchUp.js:24]: similar to above question about in-line styles. I see that the first button had the in-line initially.

**Review by PVigar88 — CHANGES_REQUESTED** [review]
Code looks good, just had a question about in-line styles being used.

File: src/components/pages/Leaderboard/RenderLeaderboard.js [hunk 3]
```diff
@@ -19,13 +19,19 @@ const RenderLeaderboard = props => {
...
+          onClick={dashboard}
+        >
+          Back to Dashboard
```
Comment (Mitch98k) [inline src/components/pages/Leaderboard/RenderLeaderboard.js:32]: Nice name change, no reason for the UI to say child or parent dashboard.

File: src/components/pages/MatchUp/RenderMatchUp.js [hunk 4]
```diff
@@ -19,7 +19,15 @@ const RenderMatchUp = props => {
...
+          className="back-btn"
```
Comment (Mitch98k) [inline src/components/pages/MatchUp/RenderMatchUp.js:25]: Good idea giving these buttons a className. If more styling is needed in the future, we can take out the inline style and put it with the class styling.

**Review by Mitch98k — CHANGES_REQUESTED** [review]
Could you insert a link to the trello card for this bug? It would really help to have more context readily available.

**Review by chrislauyc — CHANGES_REQUESTED** [review]
Good work implementing the fix. Please also attach the Trello link and a loom video going over your changes and why you made them.

**Comment by derekjpeters** [comment]
Closing PR.  There are conflicts that @felipe-sq will be working on.  Bipin has graduated so Felipe will proceed with the changes.

**Pull request closed without merging**